alf/metrics/metrics_test.py

```diff
@@ -137,16 +132,12 @@ def testMetric(self, metric_class, num_trajectories, expected_result,
                    vector_reward):
         trajectories = self._create_trajectories(vector_reward)
         if metric_class in [AverageEpisodeLengthMetric]:
-            metric = metric_class(batch_size=2)
+            metric = metric_class(example_time_step=trajectories[0])
         elif metric_class in [
-                AverageReturnMetric, AverageDiscountedReturnMetric
+                AverageReturnMetric, AverageDiscountedReturnMetric,
+                AverageEnvInfoMetric
         ]:
-
-            metric = metric_class(
-                batch_size=2, reward_shape=(2, ) if vector_reward else ())
-        elif metric_class == AverageEnvInfoMetric:
-            metric = metric_class(
-                batch_size=2, example_env_info=dict(x=0, y=0))
+            metric = metric_class(example_time_step=trajectories[0])
```

Comment:
Don't need if elif else anymore.

Reply:
Updated with `if .. else ..`. The first two branches are merged.